test/controllers/sessions_controller_test.rb

```diff
@@ -2,4 +2,28 @@
 
 # Tests for the SessionsController class
 class SessionsControllerTest < ActionController::TestCase
+  test 'old password migrates to bcrypt' do
+    post :create, session: {
+      username: users(:martin).name,
+      password: 'HODOR'
+    }
+
+    migrated_martin = User.find_by(name: users(:martin).name)
+                      .authenticate('HODOR')
+
+    assert_not_nil migrated_martin,
+                   'Could not authenticate an older user using the '\
+                   'new bcrypt scheme'
+  end
+
+  test 'user\'s old password is cleared when migrated' do
```

Comment:
RuboCop doesn't require single-quoted strings in situations like this. This felt more consistent but I also don't like escapes. I'll fix it.